src/builtins.cpp

```diff
@@ -940,11 +940,11 @@ void DefineStdlib(SymbolTable *symbolTable, llvm::LLVMContext *ctx, llvm::Module
     // Unlike regular builtins and dispatch module, which don't care about mangling of external functions,
     // so they only differentiate Windows/Unix and 32/64 bit, builtins-c need to take care about mangling.
     // Hence, different version for all potentially supported OSes.
-
-    const BitcodeLib *builtins = g->target_registry->getBuiltinsCLib(g->target_os, g->target->getArch());
-    Assert(builtins);
-    AddBitcodeToModule(builtins, module, symbolTable, warn);
-
+    if (g->target_os != TargetOS::web) {
```

Comment:
Why not executing AddBitcodeToModule(), if you need `--nostdlib` anyway (at least before the target file has complete implementation of all require intrinsics)?

Reply:
builtins.c is not compiled for wasm in this patch so Assert(builtins); fires. Maybe use Emscripten to compile it?

Reply:
Is it possible to redistribute ispc with some version of Emscripten implementation of libc? It could be compiled to wasm or LLVM IR modules and linked against at the last stage.

Reply:
Main question if emscripten license is compatible with BSD license. If it is, we can do that. Though, I'd like to avoid the dependency if possible. Probably we can have a separate implementation of print() in builtin.c for WASM, it also might be in form of LLVM IR (result of emscripten compiler). So we need a little more data before we can make an informed decision.

Reply:
I've added empty builtins.c for WASM, so it's a placeholder for future implementation.

For specific implementation we can decide after this PR is merged.